src/Lift.ts

```diff
@@ -186,26 +187,8 @@ export class Lift {
     return schemaPath
   }
 
-  public getDatamodelPath(): string {
```

Comment:
This moved to: https://github.com/prisma/lift/pull/179/files#diff-0dec20c8a33cd72e7e4efa9e513e4e4e